Sources/SWBTestSupport/CoreTestSupport.swift

```diff
@@ -50,7 +50,19 @@ extension Core {
         // When this code is being loaded directly via unit tests, find the running Xcode path.
         //
         // This is a "well known" launch parameter set in Xcode's schemes.
-        let developerPath = getEnvironmentVariable("XCODE_DEVELOPER_DIR_PATH").map(Path.init)
+        let developerPath: Path?
+        if let xcodeDeveloperDirPath = getEnvironmentVariable("XCODE_DEVELOPER_DIR_PATH").map(Path.init) {
+            developerPath = xcodeDeveloperDirPath
+        } else {
+            // In the context of auto-generated package schemes, try to infer the active Xcode.
+            let potentialDeveloperPath = getEnvironmentVariable("PATH")?.components(separatedBy: ":").first.map(Path.init)?.dirname.dirname
```

Comment:
I think that dealing with `Path` (note that the Windows spelling is `Path` rather than `PATH`) is a pretty common operation. Splitting on Windows should be on `;` not on `:`. Creating a wrapper for that would be good.

Reply:
That reminds me, forgot to file #186

Reply:
For now, please use `Path.pathEnvironmentSeparator` instead of hardcoding `:`

Reply:
Yah, I can do that, though this particular piece of code is obviously only really relevant on macOS.

Reply:
True, but just good practice IMO if used as a reference, copied elsewhere, etc.